docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1620,6 +1620,41 @@ its value will be replaced with the provided one.
 }
 --------------------------------------------------
 
+[[size-processor]]
+=== Size Processor
+Sets the size of the current document as it will be rendered as a JSON document.
+
+[[size-options]]
+.Siz Options
+[options="header"]
+|======
+| Name                  | Required | Default      | Description
+| `target`              | no       | `_meta.size` | The target field the size will be stored in
+| `consider_size_field` | no       | `false`      | Should the generated `size` field be taken into account when computing size field.
```

Comment:
I would not have this parameter in the first iteration. Maybe just document that it computes the size before adding the field? Then we'll see whether users consider it an issue